spec/descriptions/tagSyntheticResults.md

```diff
@@ -48,9 +52,27 @@ The timeFrame might be adjusted to fit the metric granularity so that there is n
 2. *pageSize* set the number of Synthetic test results you want to return with one query
 
 **order** You can order the returned items alphanumerical by label, either ascending or descending
-1. *by* Use the metric name, e.g. "response_time", to order by its value
+1. *by* Use the metric name, e.g. "synthetic.metricsResponseTime", to order by its value
```

Comment:
For aggregated test results, adding:
```
"order": {
     	"by": "synthetic.metricsResponseTime",
     	"direction": "DESC"
     }
```
got internal server error. This is the error in synthetics-reader in Pink: 
```
Caused by: com.instana.clickhouse.client.exception.ClickHouseRequestException: Code: 46. DB::Exception: Unknown function avgIf. (UNKNOWN_FUNCTION) (version 22.3.15.33 (official build))
. Query: SELECT client_id,test_id,toString((floor((start_time/600000))*600000)) AS `start_time_aggregated`,avgIf(metrics_response_time,metrics_response_time != 9223372036854775807) AS `metrics_response_time`,avgIf(metrics_status,metrics_status != 2147483647) AS `metrics_status`,metrics_response_time,groupUniqArray(location_id) FROM synthetics.all_synthetic_test_results LEFT JOIN locations_temp_table_6be1d810 ON (synthetics.all_synthetic_test_results.location_id = gk_location_id) WHERE client_id = 'saas_instana_test' AND start_time >= 1682525400000 AND start_time <= 1682532000000 AND test_id IN ('s8Cr2aTFMLUGufmd0vNK') GROUP BY client_id,test_id,start_time_aggregated,metrics_response_time,start_time_aggregated ORDER BY metrics_response_time DESC,start_time_aggregated ASC LIMIT 100 FORMAT JSON
```

Reply:
Without order by response_time or order by synthetic.metricsResponseTime, the following payload works fine: 
{
    "testId": ["s8Cr2aTFMLUGufmd0vNK"],
    "//comment1": "Get test results from last 30 minutes (windowSize), data are aggregated every 10 minutes (granularity)",
    "//comment2": "status = 1 ==> successful",
    "metrics": [
     {
        "aggregation": "MEAN",
        "granularity": 600,
        "metric": "synthetic.metricsResponseTime"
     },
     {
        "aggregation": "MEAN",
        "granularity": 600,
        "metric": "status"
     }],
     "timeFrame": {
       "to": 0,
       "windowSize": 7200000
     },
     "tagFilters":[{
     	"name": "synthetic_type",
     	"operation": "EQUALS",
     	"stringValue": "BrowserScript"
     }]
}

Reply:
I created a card to investigate further:  https://instana.kanbanize.com/ctrl_board/132/cards/125989/details/